airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeInternalStagingDestination.java

```diff
@@ -129,6 +130,11 @@ protected JdbcSqlGenerator getSqlGenerator() {
     throw new UnsupportedOperationException("Snowflake does not yet use the native JDBC DV2 interface");
   }
 
+  @Override
+  protected JdbcDestinationHandler getDestinationHandler(String databaseName, JdbcDatabase database) {
+    throw new UnsupportedOperationException("Snowflake does not yet use the native JDBC DV2 interface");
```

Comment:
after the incident with insertRecords - can you try setting up a new connector + running a sync with a dev build? just to verify that `check` still works

... I really hope getDestinationHandler isn't called anywhere on the `check` code path, but I'm scared :cry:

Reply:
Yeah absolutely. Not going to merge this without a test sync in Performance workspace.